data/human/rim jobs.txt

```diff
@@ -111,6 +118,11 @@ mission "Han Sizer Month [1]"
 	repeat
 	description `In celebration of Han Sizer month, bring <fare> to all the marked systems and return to <destination> for <payment>.`
 	passengers 2 10 .8
+	substitutions
+		"<on Longjump>" "near here."
```

Comment:
```suggestion
		"<on Longjump>" "near here"
```
Also, does that complete to `One is already uploading a travelogue to someone near here as they start walking away.` on line 145? It sound a bit weird, I think.